lib/rules/web/admin_console/4.2/cluster_settings.xyaml

```diff
@@ -0,0 +1,106 @@
+# check cluster settings tab links
+check_cluster_settings_tab_links:
+  params:
+    text: Overview 
+    link_url: /settings/cluster/
+  action: check_link_and_text
+  params:
+    text: Cluster Operators 
+    link_url: /settings/cluster/clusteroperators
+  action: check_link_and_text
+  params:
+    text: Global Configuration 
+    link_url: /settings/cluster/globalconfig
+  action: check_link_and_text
+# check version table box section
+check_table_header:
+  elements:
+  - selector:
+      xpath: //.[contains(@class, 'co-detail-table') and contains(text(), '<text>')]
```

Comment:
Why not `//*[@class='co-detail-table__section-header' and text()='<text>']`

Reply:
Checking table header is replaced by the following rule `check_resource_details_key_and_value` you suggested, updated and remove this xpath rule only for header.